From 66d2d6227c4e6a581e28b2bd6c430dee57ea8cd1 Mon Sep 17 00:00:00 2001
From: Krzysztof Chojnowski <krzysiek@embedev.pl>
Date: Thu, 19 Mar 2020 14:06:33 +0100
Subject: [PATCH] Ilitek-ili9881 fixes for openstlinux 20-02-19

---
 drivers/video/ilitek-ili9881c.c | 17 ++++++++---------
 1 file changed, 8 insertions(+), 9 deletions(-)

diff --git a/drivers/video/ilitek-ili9881c.c b/drivers/video/ilitek-ili9881c.c
index f12d0c6..3ad0402 100644
--- a/drivers/video/ilitek-ili9881c.c
+++ b/drivers/video/ilitek-ili9881c.c
@@ -3,7 +3,7 @@
 #include <common.h>
 #include <backlight.h>
 #include <dm.h>
-#include <mipi_display.h>
+#include <mipi_dsi.h>
 #include <panel.h>
 #include <asm/gpio.h>
 #include <power/regulator.h>
@@ -287,12 +287,6 @@ static int ili9881c_panel_enable_backlight(struct udevice *dev)
 	struct ili9881c_panel_priv *priv = dev_get_priv(dev);
 	int ret;
 
-	device->lanes = 2;
-	device->format = MIPI_DSI_FMT_RGB888;
-	device->mode_flags = MIPI_DSI_MODE_VIDEO |
-			     MIPI_DSI_MODE_VIDEO_SYNC_PULSE |
-			     MIPI_DSI_MODE_LPM;
-
 	ret = mipi_dsi_attach(device);
 	if (ret < 0)
 		return ret;
@@ -360,10 +354,9 @@ static int ili9881c_panel_ofdata_to_platdata(struct udevice *dev)
 static int ili9881c_panel_probe(struct udevice *dev)
 {
 	struct ili9881c_panel_priv *priv = dev_get_priv(dev);
+	struct mipi_dsi_panel_plat *plat = dev_get_platdata(dev);
 	int ret;
 
-	dev_err(dev, "Probe display\n");
-
 	if (IS_ENABLED(CONFIG_DM_REGULATOR) && priv->reg) {
 		ret = regulator_set_enable(priv->reg, true);
 		if (ret)
@@ -376,6 +369,12 @@ static int ili9881c_panel_probe(struct udevice *dev)
 	dm_gpio_set_value(&priv->reset, false);
 	mdelay(10);
 
+	plat->lanes = 2;
+	plat->format = MIPI_DSI_FMT_RGB888;
+	plat->mode_flags = MIPI_DSI_MODE_VIDEO |
+		 	   MIPI_DSI_MODE_VIDEO_SYNC_PULSE |
+			   MIPI_DSI_MODE_LPM;
+
 	return 0;
 }
 
