From 24de9efc236e5423a19450469445ba6992358e3e Mon Sep 17 00:00:00 2001
From: Krzysztof Chojnowski <krzysiek@embedev.pl>
Date: Wed, 12 Aug 2020 18:06:15 +0200
Subject: [PATCH] Boot from mmc0 only

---
 include/configs/stm32mp1.h | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/include/configs/stm32mp1.h b/include/configs/stm32mp1.h
index 7b4fe484d3..71d4861d0a 100644
--- a/include/configs/stm32mp1.h
+++ b/include/configs/stm32mp1.h
@@ -122,11 +122,7 @@
 #endif
 
 #define BOOT_TARGET_DEVICES(func)	\
-	BOOT_TARGET_MMC1(func)		\
-	BOOT_TARGET_UBIFS(func)		\
-	BOOT_TARGET_MMC0(func)		\
-	BOOT_TARGET_MMC2(func)		\
-	BOOT_TARGET_PXE(func)
+	BOOT_TARGET_MMC0(func)
 
 /*
  * bootcmd for stm32mp1:
